service/controller/key/cluster.go

```diff
@@ -16,7 +16,7 @@ import (
 const (
 	// CloudConfigVersion defines the version of k8scloudconfig in use. It is used
 	// in the main stack output and S3 object paths.
-	CloudConfigVersion = "v_6_0_0"
+	CloudConfigVersion = "v_6_1_0"
```

Comment:
Why is this still a thing? Can't we get rid of that somehow?

Reply:
We want to but this is part of the change we pushed behind HAMasters because changing this introduces conflicts with your changes.

Reply:
Buddy, everything you do here produces conflicts. It will be awful. But I will sort it out. No worries.

Reply:
BTW this is kind of my own fault because I did this big hot mess PR.

Reply:
This is part of the s3object URL so it goes into the minimal bootstrap ignition. The URL changes, so the s3object and main ignition change, so the boostrap ignition changes, so the launch configuration changes, and ultimately the workers are rolled. It all depends on whether we want workers to restart and the version defined in previous releases. I have a more elegant method for this I'll bring in after HA masters--trying to keep this as small as possible.